jaxws-retail/service/pom.xml

```diff
@@ -60,6 +61,12 @@
             <scope>provided</scope>
             <type>pom</type>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.ws.cxf</groupId>
+            <artifactId>jbossws-cxf-client</artifactId>
+            <version>${version.jbossws-cxf-client}</version>
```

Comment:
provided scope or is it supposed for this dep to be packaged in the deployment?

Reply:
Fixed